### PR TITLE
Show scrollbar from the side on wide screens

### PR DESCRIPTION
**Pull request by M-ixai-L**
https://github.com/yako-dev/flutter-settings-ui/issues/121

image

### CODE REVIEW

File: example/android/app/src/debug/AndroidManifest.xml [hunk 1]
```diff
@@ -3,5 +3,5 @@
     <!-- Flutter needs it to communicate with the running application
          to allow setting breakpoints, to provide hot reload, etc.
     -->
-    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (yadaniyil) [inline example/android/app/src/debug/AndroidManifest.xml:6]: why does the lib need internet permission?

File: example/android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```
Comment (yadaniyil) [inline example/android/gradle/wrapper/gradle-wrapper.properties:6]: not sure it's good to downgrade gradle verison, but it's ok I guess

File: example/lib/screens/gallery_screen.dart [hunk 3]
```diff
@@ -20,7 +20,9 @@ class GalleryScreen extends StatelessWidget {
             title: Text('General'),
             tiles: [
               SettingsTile.navigation(
-                title: Text('Abstract settings screen'),
+                title: Text(
```
Comment (yadaniyil) [inline example/lib/screens/gallery_screen.dart:23]: why?

**Review by yadaniyil — CHANGES_REQUESTED** [review]

**Comment by yadaniyil** [comment]
conflicts @M-ixai-L

**Review by thomas725 — APPROVED** [review]
I'd love to see this bugfix released :)

Not that I understand what this code does, I just want to see the result shown in the screenshot in the pull request @ https://github.com/yako-dev/flutter-settings-ui/pull/125 in my project's usage of your library too ;)

**Pull request merged**